kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue

```diff
@@ -140,6 +140,7 @@
           @submit="handleOpenQuiz(activeQuiz.id)"
         >
           <p>{{ openQuizModalDetail$() }}</p>
+          <p>{{ canNoLongerEditQuizNotice$() }}</p>
```

Comment:
This should be conditionalized on `quiz.draft` being `true` - otherwise we will show this warning for quizzes that couldn't be edited anyway.